### DISCUSSION TITLE
Breadcrumbs if Route is set manually

### DISCUSSION THREAD

**Discussion by HannesEURESA** [statamic/cms#10790] [Troubleshooting] [answered]
I have the Case, that i set within a Collection the Route to be shorter, than to set the {mount}
Like its shown here: https://statamic.dev/collections#routing

But now i want to get the Navigation on the Blog Article, and it´s showing me only the Main Page and the current Article, not the Blog Page itsself.

So the route is correct: 
example.com/blog/article

But the Breadcrumbs looking this way:
Main Page / Article -> the Blog is missing. 

It´s included this way: 

```
<ul class="breadcrumbs">
    {{ nav:breadcrumbs }}
    <li{{ if is_current }} class="current"{{ /if }}>
        <a href="{{ url }}">{{ title }}</a>
    </li>
    {{ /nav:breadcrumbs }}
</ul>
```

So my question is: 
Is there a way to get the Blog Page ( It´s the Mounted Page) to be visible on the breadcrumbs?

![image](https://github.com/user-attachments/assets/9f8d0e2e-a682-43f1-89a9-95ce28d578d8)

![image](https://github.com/user-attachments/assets/6cc4f5a2-2fe4-44e2-bcf0-e9720ad0ee4e)

**Comment by edalzell** [comment, marked as answer]
Is this a valid route: `example.com/blog`?

**Reply by HannesEURESA** [reply]
Thanks a lot. That was the case. I created the Menue as Alias and do not display under the Menue. 
Normally I have this structure:
- blogtopics/blog/article -> But I don´t wanted the Breadcrumb to be that long. so I cutted the mount to /blog/{slug}. So the Menu Blog exists, but not as "Alias" in the first Segment. It is in the Child Segment, so thats maybe? But anyways, good to know if I create an Alias it will work.
